shared/src/commonMain/composeResources/values/strings.xml

```diff
@@ -12,6 +12,8 @@ We will process your data in accordance with the App Privacy Policy. You can adj
     <string name="notifications_description">Customize your preferences to ensure you receive only the notifications that matter most to you, avoiding unnecessary disruptions.</string>
     <string name="notifications_lets_get_started">Let’s get started!</string>
 
+    <string name="code_of_conduct">Code of conduct</string>
```

Comment:
```suggestion
    <string name="code_of_conduct">Code of Conduct</string>
```

I see this spelling in both the design and the content doc

Reply:
The casing is inconsistent in design mockups, asked for clarification